src/test/kotlin/no/nav/familie/ba/mottak/util/IdentUtilsTest.kt

```diff
@@ -8,21 +8,18 @@ import org.junit.jupiter.api.Test
 class IdentUtilsTest {
     @Test
     fun testErDnummer() {
-        Assertions.assertFalse(erDnummer(PersonIdent(id = null)))
         Assertions.assertFalse(erDnummer(PersonIdent(id = "12345678910")))
         Assertions.assertTrue(erDnummer(PersonIdent(id = "42345678910")))
     }
 
     @Test
     fun `er BOST-nummer`() {
-        Assertions.assertFalse(erBostNummer(PersonIdent(id = null)))
         Assertions.assertFalse(erBostNummer(PersonIdent(id = "10102022222")))
         Assertions.assertTrue(erBostNummer(PersonIdent(id = "10201022222")))
     }
 
     @Test
     fun `er FDAT nummer`() {
-        Assertions.assertFalse(erFDatnummer(PersonIdent(id = null)))
         Assertions.assertFalse(erFDatnummer(PersonIdent(id = "10102022222")))
         Assertions.assertTrue(erFDatnummer(PersonIdent(id = "10201000000")))
```

Comment:
Ikke relatert til dine endringer, men: Det hadde vært fint med en ekstra test på Fdat-nummer som slutter på 1, spesielt siden det gjøres en sjekk på om barnet har Fdat-nummer når man mottar fødselshendelse. Fdatnummer som slutter på 1 betyr dødfødt barn, så det er et reellt case for fødsler som vi bør håndtere :)